vsintegration/src/FSharp.Editor/Common/LanguageService.fs

```diff
@@ -233,7 +233,12 @@ type internal FSharpLanguageService(package : FSharpPackage) as this =
         let site = siteProvider.GetProjectSite()
         let projectGuid = Guid(site.ProjectGuid)
         let projectFileName = site.ProjectFileName()
-        let projectId = workspace.ProjectTracker.GetOrCreateProjectIdForPath(projectFileName, projectFileName)
+
+        let projectDisplayName = 
+            if String.IsNullOrWhiteSpace projectFileName then projectFileName
```

Comment:
````
if String.IsNullOrWhiteSpace projectFileName then projectFileName 
````
This makes the display name empty if projectFileName is empty.  I suppose this is as good as any ... but it displaying an empty string there seems a bit weak.